### ISSUE TITLE
"Failed to produce item" error

### ISSUE THREAD

**Issue by gabrielcyrino** [getnamo/myo-ue4#9]
Hello there,
I'm having some issues when packaging my project. I am using pugin version 0.9.1 and UE 4.19.2. When trying to packaging for Windows x64, the following error occurs:
`ERROR: UBT ERROR: Failed to produce item: C:\UEProjects\HarpyGame\Plugins\myo-ue4\Binaries\Win64\UE4-MyoPlugin.lib`
I tried to use the last plugin release doing full rebuild in the project. I tried to insert the property `"Installed": true` in the .uplugin file. I also tried to add the line `EnabledPlugins = MyoPlugin` in DefaultEngine.ini, and other attempts. Nothing worked.
Thank you in advance.

Log:
```
uprojectpath is: C:\UEProjects\HarpyGame
Parsing headers for HarpyGame
  Running UnrealHeaderTool "C:\UEProjects\HarpyGame\HarpyGame.uproject" "C:\UEProjects\HarpyGame\Intermediate\Build\Win64\HarpyGame\Development\HarpyGame.uhtmanifest" -LogCmds="loginit warning, logexit warning, logdatabase error" -Unattended -WarningsAsErrors -installed
Reflection code generated for HarpyGame in 7,4045565 seconds
Performing 24 actions (4 in parallel)
[3/24] Resource PCLaunch.rc
PCH.UELinkerFixups.cpp
[5/24] Resource PCLaunch.rc
SharedPCH.Engine.cpp
PCH.MyoPlugin.cpp
UELinkerFixups.cpp
GameDBController.cpp
GameDBController.gen.cpp
DBPersistence.cpp
HarpyGameGameModeBase.gen.cpp
HarpyGameGameModeBase.cpp
UnrealMosquitto.cpp
UnrealMosquitto.gen.cpp
MQTTMessage.gen.cpp
Desafio.gen.cpp
HarpyGameFunctionLibrary.gen.cpp
MQTTMessage.cpp
HarpyGame.init.gen.cpp
HarpyGameFunctionLibrary.cpp
HarpyGame.cpp
C:\UEProjects\HarpyGame\Source\HarpyGame\HarpyGameFunctionLibrary.cpp(27): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
C:\UEProjects\HarpyGame\Source\HarpyGame\HarpyGameFunctionLibrary.cpp(28): warning C4996: 'fopen': This function or variable may be unsafe. Consider using fopen_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
Module.MyoPlugin.cpp
Module.MyoPlugin.gen.cpp
ERROR: UBT ERROR: Failed to produce item: C:\UEProjects\HarpyGame\Plugins\myo-ue4\Binaries\Win64\UE4-MyoPlugin.lib
Total build time: 51,81 seconds (Local executor: 0,00 seconds)
```

**Comment by getnamo** [comment]
Have you tried cleaning your project, deleting intermediate folders and rebuilding? I'm not seeing any Myo specific errors so it would be hard to debug. Consider compiling the project for e.g. Development mode in C++ and see if it spits out any more detailed errors.

Otherwise only errors shown are from your own game function library, use sprintf_s and fopen_s if possible to mute errors.

**Comment by gabrielcyrino** [comment]
Yes, solved.

I cleaned and compiled the project in Developer mode in C++ and I could see the error in detail:

`C4530: C++ exception handler used, but semantics are not enabled. Specify /EHsc`

This error is due to try/catch statements that were used. I fixed it by adding the line `bEnableExceptions = true` inside the {ProjectName}.Build.cs constructor.

This error also appeared in the Hub_impl.hpp file, witch seems to belong to myo plugin. So I had to add the same line inside the MyoPlugin.Build.cs constructor.

Thanks a lot for the help!
